tests/config_test.go

```diff
@@ -145,7 +153,12 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			BeforeEach(func() {
 				for i := 0; i < configMapsCnt; i++ {
 					name := "configmap-" + uuid.NewString()
-					tests.CreateConfigMap(name, testsuite.GetTestNamespace(nil), map[string]string{"option": "value"})
+					_, err := kubevirt.Client().CoreV1().ConfigMaps(testsuite.GetTestNamespace(nil)).Create(context.Background(), &k8sv1.ConfigMap{
+						ObjectMeta: metav1.ObjectMeta{Name: name},
+						Data:       map[string]string{"option": "value"},
+					}, metav1.CreateOptions{})
+
+					Expect(err).NotTo(HaveOccurred())
```

Comment:
I'd swap order of empty line in Expect, so Expect comes immediately after the assignment to `err`, and then a new line.

Reply:
@0xFelix should make changes to all file or just in `config_test.go`